build.xml

```diff
@@ -119,11 +120,22 @@
         </condition>
     </target>
 	
-	<target name="jarSkipTests">
+	<target name="jarSkipTests" description="create jar file skip unit tests">
 		 <property name="RUN_NO_TESTS" value="Yes"/>
 		<antcall target="jar"/>
 	</target>
 
+	<target name="jarSkipTestAndSubprojectBulds" description="create jar file skip unit tests do not build sub projects">
+		 <property name="RUN_NO_TESTS" value="Yes"/>
+		 <property name="BUILD_NO_SUBPROJECTS" value="Yes"/>
+		<antcall target="jar"/>
+	</target>
+
+	<target name="jarSkipBuildSubprojectsOnly" description="create jar file, run unit tests, skip building sub projects ">
+		 <property name="BUILD_NO_SUBPROJECTS" value="Yes"/>
```

Comment:
I think it's very confusing to have the word "Only" in this target name.   Another test (above) is `jarSkipTests`; I think an analogous (better) name for this target would be `jarSkipBuildSubprojects`.  That also makes it pretty consistent with `jarSkipTestsAndSubprojectBuilds`...

Reply:
pushed fix.